scripts/dataset_processing/ljspeech/extract_ljspeech_phonemes_and_durs.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# Extracts the phonemes and alignments for the LJSpeech dataset via the
+# Montreal Forced Aligner (MFA) library, and computes durations per phoneme.
+# Assumes you have downloaded and expanded the LJSpeech dataset, and that they
+# are located at the directory specified.
+# Also requires that you have manifests set up pointing to the appropriate
+# train/val/test wav files. Make sure you have run the
+# `create_manifests_and_textfiles.py` script.
```

Comment:
I wasn't able to get the script/conda env to play well with the NeMo installation/importing parsers; for some reason it could never find the installation. If you can get it to work or have any tips, though, I'd be happy to add it in and make everything less clunky.

Reply:
I see. Let's just update the README to say to run `create_manifests_and_textfiles.py` first, then the `extract_ljspeech_phonemes_and_durs.sh`

Reply:
Ah yep, updated.